client/rpc_sender.go

```diff
@@ -83,6 +84,13 @@ func newRPCSender(server string, context *base.Context, retryOpts retry.Options,
 // and been executed successfully. We retry here to eventually get through with
 // the same client command ID and be given the cached response.
 func (s *rpcSender) Send(ctx context.Context, ba roachpb.BatchRequest) (*roachpb.BatchResponse, *roachpb.Error) {
+	timeout := 5 * s.retryOpts.MaxBackoff
```

Comment:
We could move the default retry options into separate files with build tags so that only race builds get the increased timeouts. Or maybe we could increase the InitialBackoff only for tests that also use failfast.